bsp/thead-smart/applications/main.c

```diff
@@ -0,0 +1,35 @@
+
+/*
+ * File      : main.c
+ * This file is part of RT-Thread RTOS
```

Comment:
Please provide corrent file header for copy right information as Apach License v2.0